src/components/com_tjcertificate/site/views/certificate/view.html.php

```diff
@@ -70,13 +70,13 @@ public function display($tpl = null)
 		{
 			$certificate = TJCERT::Certificate();
 			$this->certificate = $certificate::validateCertificate($this->uniqueCertificateId);
+		}
 
-			if (!$this->certificate->id)
-			{
-				JError::raiseWarning(500, Text::_('COM_TJCERTIFICATE_ERROR_CERTIFICATE_EXPIRED'));
+		if (!$this->certificate->id)
+		{
+			JError::raiseWarning(500, Text::_('COM_TJCERTIFICATE_ERROR_CERTIFICATE_EXPIRED'));
 
-				return false;
-			}
+			return false;
```

Comment:
Don't return false, it won't show me the search box again. Also, did you check if the image generation code is running if the certificate is not present @tushar33

Reply:
done